proposals/0292-package-registry-service.md

```diff
@@ -788,6 +790,17 @@ A registry can further improve on this model by implementing a
 or another comparable, tamper-proof system
 for authenticating package contents.
 
+Distribution of packages through Zip files
+introduces new potential attack vectors.
+For example,
+an attacker could maliciously tamper with a generated source archive
+in an attempt to exploit
+a known vulnerability like [Zip Slip],
+or a common software weakness like susceptibility to a [Zip bomb].
+Swift Package Manager should take care to
+identify and protect against these kinds of attacks 
+in its implementation of source archive decompression.
```

Comment:
@robertlacroix What we have right now with `ZipArchiver` delegates to `unzip`:  

https://github.com/apple/swift-tools-support-core/blob/435a2708a6e486d69ea7d7aaa3f4ad243bc3b408/Sources/TSCUtility/Archiver.swift#L65

Right now, we're using this only for user-specified source files in [binary targets](https://github.com/apple/swift-evolution/blob/main/proposals/0272-swiftpm-binary-dependencies.md). Do you think we should do anything differently in support of archives sent from the package registry?